arrow/__init__.py

```diff
@@ -15,3 +15,22 @@
     FORMAT_W3C,
 )
 from .parser import ParserError
+
+__all__ = [
```

Comment:
Mind adding a comment documenting why this is necessary for mypy? Maybe link to the section in the mypy docs?

Reply:
Thanks, that slipped my attention.